ui/app/templates/vault/cluster/access/mfa/enforcements/enforcement/index.hbs

```diff
@@ -21,30 +21,51 @@
 <div class="tabs-container box is-sideless is-fullwidth is-paddingless is-marginless">
   <nav class="tabs" aria-label="Enforcement tabs">
     <ul>
-      <LinkTo @route="vault.cluster.access.mfa.enforcements.enforcement" @query={{hash tab="targets"}}>
+      <LinkTo
+        @route="vault.cluster.access.mfa.enforcements.enforcement"
+        @query={{hash tab="targets"}}
+        data-test-tab="targets"
+      >
         Targets
       </LinkTo>
-      <LinkTo @route="vault.cluster.access.mfa.enforcements.enforcement" @query={{hash tab="methods"}}>
+      <LinkTo
+        @route="vault.cluster.access.mfa.enforcements.enforcement"
+        @query={{hash tab="methods"}}
+        data-test-tab="methods"
+      >
         Methods
       </LinkTo>
     </ul>
   </nav>
 </div>
 <Toolbar>
   <ToolbarActions>
-    <button class="toolbar-link" onclick={{action (mut this.showDeleteConfirmation) true}} type="button">
+    <button
+      class="toolbar-link"
+      onclick={{action (mut this.showDeleteConfirmation) true}}
+      type="button"
+      data-test-enforcement-delete
+    >
       Delete
     </button>
     <div class="toolbar-separator"></div>
-    <ToolbarLink @params={{array "vault.cluster.access.mfa.enforcements.enforcement.edit" this.model.id}}>
+    <ToolbarLink
+      @params={{array "vault.cluster.access.mfa.enforcements.enforcement.edit" this.model.id}}
+      data-test-enforcement-edit={{true}}
+    >
       Edit enforcement
     </ToolbarLink>
   </ToolbarActions>
 </Toolbar>
 
 {{#if (eq this.tab "targets")}}
   {{#each @model.targets as |target|}}
-    <LinkedBlock class="list-item-row" @disabled={{not target.link}} @params={{union (array target.link) target.linkModels}}>
+    <LinkedBlock
+      class="list-item-row"
+      @disabled={{not target.link}}
+      @params={{union (array target.link) target.linkModels}}
```

Comment:
TIL "union" Never seen that before.

Reply:
First time I've used it myself. It's a part of [ember-composable-helpers](https://github.com/DockYard/ember-composable-helpers#union)